tenant/README.md

```diff
@@ -0,0 +1,32 @@
+# Tenant
+
+This folder provides a set of CRDs used to manage tenant resources in a Kubernetes cluster.
+
+## Overview
+A cluster admin primarily leverages a tenant admin namespace to create, and thus manage all tenant related API objects including tenant namespaces (not directly though), arbitrary customized CRDs and policy objects like RBAC. This makes the tenant concept fully extensible instead of limiting the types of tenant objects specifically in TenantSpec. Basically, the tenant admin namespace name is specified in Tenant CR and tenant controller simply creates the namespace and configure the cluster RBAC rules for tenant admins to access various tenant related objects which are created on-demand (not implmented yet).
+
+The most common tenant objects are tenant namespace CRs which encapsulate the configurations of tenant namespaces including name and RBAC rules etc. Tenant namespace controller does actual namespace creation for tenant users to use. The primary reason why we introduce tenant namespace CRD is that now tenant namespace creation can be RBAC permissioned, and tenant admins can do `kubectl get tenantnamespace -n <tenant admin namespace>` to list all the tenant namespaces. Otherwise had K8s namespace list API been used, all namespaces in the cluster would be exposed to tenant admins. Tenant admin can do self-service namespace creation via creating tenant namespace CR or even import existing namespace to tenant management system.
+
+## Getting Started
+
```

Comment:
I think it would be nice to include the basic setup needed to try this : 
1. Install Go
2. Install docker
3. Install KIND and kind create cluster

Reply:
I'd rather put these specific steps to a tutorial document instead of README.  People may try this successfully in their own environments without these steps.